broker/cloud_functions/lvk/store_BigQuery/deploy.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+# Deploys or deletes broker Cloud Function
+# This script will not delete Cloud Functions that are in production
+
+testid="${1:-test}"
+# "False" uses production resources
+# any other string will be appended to the names of all resources
+teardown="${2:-False}"
+# "True" tearsdown/deletes resources, else setup
```

Comment:
These comments seem improperly interleved with the variables they are documenting.

Reply:
They are, and this is a holdover of a mistake I made years ago.